android/app/src/googleplay/java/org/fossasia/openevent/fragments/MapsFragment.java

```diff
@@ -124,7 +118,29 @@ private void showLocationsOnMap() {
             builder.include(marker.getPosition());
         }
 
-        handleSearchEvents();
+        LayoutInflater inflator = (LayoutInflater) getActivity()
```

Comment:
@iamareebjamal I have made all the changes in the previous commits, can you tell please which one?